charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus.yaml

```diff
@@ -222,6 +201,12 @@ spec:
               - key: bundle.crt
                 path: ca.crt
               optional: false
+      - name: ca-etcd
```

Comment:
The PR title is drop legacy client certificate for Prometheus, but here we are also making changes related to projected volume mounts. Is there a need to mention this in the title? Otherwise
/lgtm

Reply:
The main purpose of this PR is dropping the client cert, the other changes are just required adaptations